test/async-hooks/test-callback-error.js

```diff
@@ -62,11 +62,12 @@ assert.ok(!arg);
   let program = process.execPath;
   let args = [
     '--abort-on-uncaught-exception', __filename, 'test_callback_abort' ];
-  const options = { encoding: 'utf8' };
+  let options = { encoding: 'utf8' };
   if (!common.isWindows) {
-    program = `ulimit -c 0 && exec ${program} ${args.join(' ')}`;
+    [program, options] = common.escapePOSIXShell`ulimit -c 0 && exec "${program}" ${args[0]} "${args[1]}" ${args[2]}`;
```

Comment:
`options`'s re-assignment here is easily missed

Reply:
I mean, that's what `let` is suppose to convey, that the variabled gets reassigned.

Reply:
Or a mistake, or missed in an update. I very nearly missed it thinking you had made a larger change, updated your code, and forgot to restore the const.

Not a big deal

Reply:
> forgot to restore the const.

The linter would catch that.